CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @anabrunner @befunger @CorreyL @CSTSTEVENU @garenpham @immangat @nam-m @openBC-ca @PooriaT @SamHuo213

```

Comment:
@SamHuo213 Note that the errors on highlight here:

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/c4da0f36-8778-4a7b-9854-d2a0945d8d0b)

Just indicate that we need to add those members with errors to this repo with `Write` access.

Upon confirmation from you, I can go ahead and do that.